includes/class-config.php

```diff
@@ -301,39 +301,42 @@ public function init_predefined_pages() {
 		 * @since 1.0
 		 */
 		public function init_email_notifications() {
-			$this->email_notifications = array(
```

Comment:
see filter on the line 98, we don't need separate filters inside each init function